README.md

```diff
@@ -286,11 +286,6 @@ bin/zxp config _config-file_
 ## Enable Metrics
 In the zot with all extensions case see [configuration example](./examples/config-metrics.json) for enabling metrics
 
-# Caveats
```

Comment:
Did not saw this PR before commenting on #384 : no need to keep track of Go compatibility version anymore